src/databaseTest/java/net/sf/jabref/shared/DBMSSynchronizerTest.java

```diff
@@ -43,15 +42,14 @@
     @Before
     public void setUp() throws ClassNotFoundException, SQLException, DatabaseNotSupportedException {
 
-        Globals.prefs = JabRefPreferences.getInstance();
-
         connection = TestConnector.getTestConnection(dbmsType);
 
         bibDatabase = new BibDatabase();
         BibDatabaseContext context = new BibDatabaseContext(bibDatabase);
 
 
-        dbmsSynchronizer = new DBMSSynchronizer(context);
+        dbmsSynchronizer = new DBMSSynchronizer(context,
+                JabRefPreferences.getInstance().get(JabRefPreferences.KEYWORD_SEPARATOR));
```

Comment:
I would prefer a hard-coded keyword separator (instead of taking the one from the preferences) here and in the next tests.